### PR TITLE
Created schema for the RDS

### PR DESCRIPTION
**Pull request by ebradley12**
Ticket #18

Created schema for the RDS: Implemented a relational database schema in Amazon RDS to structure and store data for the Band Camp Tracker project. The schema includes normalised tables for sales, customers, artists, releases, genres, and subscribers, with relationships defined using primary and foreign keys to maintain data integrity.

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    FOREIGN KEY (customer_id) REFERENCES customer(customer_id),
+    FOREIGN KEY (release_id) REFERENCES release(release_id)
+);
+
+
+CREATE TABLE subscriber (
+    subscriber_id BIGINT PRIMARY KEY,
+    subscriber_email VARCHAR UNIQUE NOT NULL,
+    alerts BOOLEAN DEFAULT FALSE,
+    CHECK (subscriber_email ~* '^[A-Za-z0-9._%+-]+@[A-Za-z0-9.-]+\.[A-Za-z]{2,}$')
```
Comment (GabrielNsiah) [inline schema.sql:64]: Is it possible to use the full regex for emails here:
```bash
^((([!#$%&'*+\-/=?^_`{|}~\w])|([!#$%&'*+\-/=?^_`{|}~\w][!#$%&'*+\-/=?^_`{|}~\.\w]{0,}[!#$%&'*+\-/=?^_`{|}~\w]))[@]\w+([-.]\w+)*\.\w+([-.]\w+)*)$
```

Comment (ebradley12) [reply schema.sql:64]: According to google, when working with SQL it's better to use a simpler regex. Apparently it's better suited for database constraints and overly strict regex can lead to false negatives, rejecting emails that are valid but uncommon

**Review by emily-curtis — APPROVED** [review]
looks good!

**Review by GabrielNsiah — APPROVED** [review]
Looks good to me.

**Pull request merged**